README.md

```diff
@@ -4,11 +4,15 @@ A basic BACnet IP server example written in C# using the [CAS BACnet Stack](http
 
 ## Releases
 
-Build versions of this example can be downloaded from the [Releases](https://github.com/chipkin/BACnetServerExampleCSharp/releases) page.
+Build versions of this example can be downloaded from the [Releases](https://github.com/chipkin/BACnetServerExampleCSharp/releases) page. 
 
 ## Installation
 
 Download the latest release zip file on the [Releases](https://github.com/chipkin/BACnetServerExampleCSharp/releases) page.
+Copy CASBACnetStack_x64_Release.dll from the stack into the release folder and launch server by using the following command:
```

Comment:
See the notes from  https://github.com/chipkin/BACnetClientExampleCSharp/pull/8

Copy CASBACnetStack_x64_Release.dll from the CAS BACnet Stack into the release folder. Please contact Chipkin Automation System for access to the CAS BACnet Stack. Launch client by using the following command: